### ISSUE TITLE
Package broken: Kijiji-scraper returns no results

### ISSUE THREAD

**Issue by limziwen** [mwpenny/kijiji-scraper#69]
When running the kijiji-parser library with these params and options:
```
{
  locationId: 1700273,
  categoryId: 42,
  sortByName: 'dateDesc',
  ll: '43.6629,-79.3957',
  distance: 3
}
{ minResults: 3 }
```
`let listings = await kijiji.search(params, options);`

We see there are no results returned. We find the same issue when toggling with various parameters for locationId, ll, distance

we suspect that Kijiji could be preventing these requests, going through. Can you invesitgate this?

**Comment by limziwen** [comment]
@malavv Could you provide us an update on this situation? Seems like this is feature breaking

**Comment by mwpenny** [comment]
Hello, thank you for the report and reproduction case. I can look at this tomorrow.

**Comment by mwpenny** [comment]
Is there any difference if you include `scraperType: "html" ` in `options`? Note that when trying this, some parameter names will  differ. See https://github.com/mwpenny/kijiji-scraper#arguments-3

**Comment by mwpenny** [comment]
I am using the following sample code based on your reproduction case:
```js
const params = {
    locationId: 1700273,
    categoryId: 42,
    sortByName: "dateDesc",
    ll: "43.6629,-79.3957",
    distance: 3
};

const options = {
    minResults: 3
};

kijiji.search(params, options).then((ads) => {
	ads.forEach(ad => console.log(`[${ad.id}] ${ad.title}`));
});
```

I get the following output:
> [1671158554] FURNISHED APARTMENTS . WITH WIFI.TV. NO LEASE. MANY LOCATIONS
> [1671129599] Furnished room for female/females
> [1671117341] A  ROOM FOR RENT
> [1671108760] Very large Luxurious 1 bedroom
> [1671057386] Short Term Shared  Room Rental Scarborough
> [1671017198] 1 bedroom basement for couple in scarborough
> [1670768212] For Rent
> [1670644257] Room for rent in Scarborough - 1 month tenancy (September)
> [1670550338] Short term rental okay: New comers Okay.
> [1670508340] Furnished condo all utilities included
> [1670468379] Luxury condo in scarborough centre for short term rental
> [1670263808] Short Term Rental Daily or Weekly. Licensed by City of Toronto
> [1669956606] 2 bhk for rent
> [1663024928] FURNISHED BUNGALOW BASMENT FOR RENT
> [1669785400] Temporary accommodation
> [1669607820] Students / Couples / New Immigrants until 30 Nov Only
> [1669451683] Room For Rent
> [1669215889] Cottage for Rent - Balm Beach (Tiny ON) Beach Walking Distance
> [1669018717] Looking for shared accommodation for 2
> [1668678774] Accommodation for September

Are you using the latest version of the module?

**Comment by mwpenny** [comment]
Looking again at your original sample code, I can see you are definitely using an old version. `sortByName`, for example, is only valid when using HTML-based scraping. This is the technique kijiji-scraper used for quite a while until it became less reliable. Versions as of 6.0.0 use the Kijiji mobile API by default. As a result, some property names are different. For example, sorting is specified using the `sortType` parameter.

Even on newer versions though you can switch back to HTML-based scraping using the `scraperType` option that I mentioned above. The fact that this option is missing while your sample code contains the old property name shows you are not up to date.

This is why I was initially unable to reproduce the problem. The module defaults to API-based scraping which works fine. When I added `scraperType: "html"` I indeed saw no results. I've fixed the HTML-based search code in PR #70 and the updated version is now on NPM.

So to address your issue you will need to update to 6.3.3 and either switch your search parameters to ones that work with API-based scraping (see the readme) or use `scraperType: "html"`. I recommend the former since it's more reliable.

Given:
1. The lack of a response
2. A new version exists with a fix for HTML-based searching that I've tested to work with your use case
3. The fact that your issue can be solved in 2 ways based on the instructions above

I'm going to close this. Feel free to open another issue if you still have problems with the new version.
